test/test_manifest.json

```diff
@@ -1191,6 +1191,12 @@
        "link": true,
        "type": "eq"
     },
+    {  "id": "Highlight.without.appearance.stream",
```

Comment:
Same here about renaming to the same as the file name and putting it below the existing highlight test. Similar for the other three files.